✨ feat (Posts): Refractor dateFormatter for AR and FR languages

mbayedione10 opened the pull request.

dofbi commented:
Hello @mbayedione10 
Good job 👍🏾  Build ok review ok si tu peux juste faire un Git Squash
https://www.gitkraken.com/learn/git/git-squash

Et refaire un push en force

mbayedione10 commented:
Hello @dofbi 
Je viens de update avec un Git Squash 👌🏿

dofbi commented:
Salaam @mbayedione10 
Si tu as fait ton sqash sur ta machine tu as un commit. Que tu puch sur ton github avec --force pour écraser les anciens commit. Si ton PR n'a pas un commit recréer le

mbayedione10 committed:
✨ feat (Mediatheque): Create and dateFormatter for AR and FR

✨ feat (Equipe): delete console

✨ feat (Mediatheque): Create Grid components for embed youtube videos


✨ feat (Mediatheque): Create Mediaatheque Page and retrieve data


✨ feat (Mediatheque): Add translation for mediatheque description

mbayedione10 commented:
Salam @dofbi 
J'avais fais le squash par tache
Le 2eme squash est sur une autre tache (créer la page mediatheque)
J'ai update le squash comme demandé
